todo_app/task/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import path
+
+from .views import (
+    CreateTask,
+    HomeView,
+    EditTaskView,
+    DeleteTask,
+)
+
+urlpatterns = [
+    path("create/", CreateTask.as_view(), name="create-task"),
+    path("home/", HomeView.as_view(), name="home"),
+    path("edit/<int:task_id>/", EditTaskView.as_view(), name="edit-task"),
+    path("delete-task/<int:task_id>/", DeleteTask.as_view(), name="delete-task"),
+]
```

Comment:
all urls should be in same file as all of these belong to the same app. `todo_app/api/urls.py`

Reply:
Done that